.github/actions/bot/index.js

```diff
@@ -10,6 +10,8 @@ async function bot(core, github, context, uuid) {
     }
     console.log("Comment found in payload");
 
+    // user's org membership must be public for the author_association to be MEMBER
+    // go to the org's member page, find yourself, and set the visibility to public
     const author = payload.comment.user.login;
     const authorized = ["OWNER", "MEMBER"].includes(payload.comment.author_association);
```

Comment:
just curious would we ever want `"COLLABORATOR"`?

Reply:
We wouldn't, because we can't expose the CI creds to any third parties. Github has a pretty good blog post covering this type of issue, it's messy: https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

Reply:
👍 the examples in that blog are nice too

besides permissions on chore jobs like comments it lgtm